sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableClient.java

```diff
@@ -244,89 +241,123 @@ public void updateEntity(Entity entity, boolean ifUnchanged, UpdateMode updateMo
      * @return a response
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Response<Void> updateEntityWithResponse(Entity entity, boolean ifUnchanged, UpdateMode updateMode,
-        Duration timeout, Context context) {
+    public Response<Void> updateEntityWithResponse(TableEntity entity, boolean ifUnchanged, UpdateMode updateMode,
+                                                   Duration timeout, Context context) {
```

Comment:
I'm usually not a fan of aligning parameters, because we have some seriously long method/class names and the parameters get mushed against the right margin.

Reply:
Fair enough, I don't have a strong opinion, so I just went with whatever IntelliJ did by default. If we decide to care more about this we can probably encode it in an editorconfig (or more likely an intellij settings file, since this might be IDE-specific).